AsyncDisplayKit/ASCellNode.m

```diff
@@ -134,6 +133,11 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event
   [(_ASDisplayView *)self.view __forwardTouchesCancelled:touches withEvent:event];
 }
 
+- (void)_visibleNodeDidScroll:(UIScrollView *)scrollView withCellFrame:(CGRect)cellFrame
```

Comment:
rename all instances of this to not have _.  now that we support the subclass override thing, we should make this method selector public and no _.